docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -83,60 +83,76 @@ The most commonly used CR settings are listed in this table:
 |Description
 
 |`cache_data_volume_size`
-|100 GiB
+|`100 GiB`
```

Comment:
I know you didn't write it, but consider changing line 74 to read "To work directly..." The "If clause" is a bit fluffy IMHO.

Reply:
Changed